README.md

```diff
@@ -1,2 +1,17 @@
 # commit.dev
+
 Marketing website for Commit (commit.dev)
+
+## Getting Started
+
+```
+git clone https://github.com/commitdev/commit.dev.git
+```
+
+```
+npm install
+```
+
+```
+npm start
+```
```

Comment:
I'd suggest grouping these lines together. No real benefit from having them in their own blocks. I'd also suggest using the dev script rather than the start script (which is meant for production and requires a production build to have been done). Can do the following instead:
```
git clone https://github.com/commitdev/commit.dev.git
npm install
npm run dev
```

Reply:
Hi, @jasondippel I'll have these changes implemented right away.

Reply:
it should probably be `npm start` too -- `npm run dev` should be used for contributors (and not someone just trying to run the project in general).

Reply:
@superkhau If `npm start` is used, we need to also run `npm run build` (otherwise start fails with an error saying it can't find a production build). I think including the `npm run dev` step instead is simpler and gets it up and running for people a little faster. I also think that the majority of people attempting to run this repo will be people looking to contribute to it and thus will want to run it in dev mode. 
In my head, the "Getting Started" section is geared towards developers. If this was a project that people would expect to run and then use, then I would suggest the "Getting Started" section be how to run the production version of the app.

Reply:
@superkhau so which method do you prefer in the readme in the fourth step?

1) `git clone https://github.com/commitdev/commit.dev.git`
2) `cd commit.dev`
3) `npm install`
4) `npm start` OR `npm run dev`

Also, @jasondippel is correct in saying that we do need to run `npm run build` before `npm start`. 
Please let me know how you would prefer the README.md to look like.

Thank you

Reply:
@jasondippel Good points, let's go with dev based on your suggestions above. ;)